src/ec/packages/ec-component-breadcrumb-core/ecl-breadcrumb-core.html.twig

```diff
@@ -21,11 +21,14 @@
         ...
       ]
     - "_compliance" (boolean) (default: false) Activates debug
```

Comment:
sometimes it's **__compliance__** with double underscore sometime not?

Reply:
it's a special parameter, that's why this idea of using this syntax with the "double" underscore , to make it stand out, anyway the parameter is `_compliance_`, i need to correct this.